token/core/cmd/pp/common/common.go

```diff
@@ -68,3 +87,54 @@ func SetupIssuersAndAuditors(pp PP, Auditors, Issuers []string) error {
 	}
 	return nil
 }
+
+func ReadFile(file string) ([]byte, error) {
+	fileCont, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not read file %s", file)
+	}
+
+	return fileCont, nil
+}
+
+func ReadPemFile(file string) ([]byte, error) {
+	bytes, err := ReadFile(file)
+	if err != nil {
+		return nil, errors.Wrapf(err, "reading from file %s failed", file)
+	}
+
+	b, _ := pem.Decode(bytes)
+	if b == nil { // TODO: also check that the type is what we expect (cert vs key..)
+		return nil, errors.Errorf("no pem content for file %s", file)
+	}
```

Comment:
Here you just want to check if `bytes` contains at least a valid pem decoded block, right? What is multiple blocks are available?